org.jacoco.core.test/src/org/jacoco/core/test/validation/ValidationTestBase.java

```diff
@@ -120,13 +120,15 @@ public void execute_assertions_in_comments() throws IOException {
 	private void assertCoverage(final Line line, final int insnStatus,
 			final int missedBranches, final int coveredBranches) {
 		final ILine coverage = line.getCoverage();
+		final String location = String.format("(%s:%d)",
```

Comment:
What about making `Line.toString()` return `<sourcename>:<nr>`?

Reply:
Then all information can be extracted from the `Line` instance passed to the assert methods.

Reply:
Done